NEWS.md

```diff
@@ -1,3 +1,8 @@
+# noctua 1.8.1.9000
+## New Feature
+* Move `sql_escape_date` into `dplyr_integration.R` backend (#121). Thanks to @OssiLehtinen for developing Athena date translation.
```

Comment:
The hyperlink to # 121 has to be set up like: `(RAthena: [# 121](https://github.com/DyfanJones/RAthena/issues/121))`. The reason for this is that github will look for issue 121 on `noctua`'s github on and not RAthena 😄

Reply:
Ah sorry about that, did a lazy copy paste there...